web-app/src/components/Polaroid/Polaroid.scss

```diff
@@ -16,4 +17,9 @@
 
 .Polaroid img {
     width: 100%;
-}
\ No newline at end of file
+    user-select: none;
+    -moz-user-select: none;
+    -webkit-user-drag: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+}
```

Comment:
There are all fallback for browser, but there's one that doesn't fit in this list: ```-webkit-user-drag: none;```
Also, I guess that the fallbacks are auto generated by sass (correct me if 'm wrong)